models/seed-models.js

```diff
@@ -49,20 +49,47 @@ exports.fetchUsers = () => {
   });
 };
 
-exports.fetchArticles = () => {
-  return db
-    .query(
-      `
+exports.fetchArticles = (sort_by = 'created_at', order_by = 'DESC', topic) => {
+   
+    const validSortBys = ['article_id', 'title', 'body', 'author', 'topic', 'created_at', 'votes', 'comment_count']
+    const validOrderBys = ['ASC', 'DESC']
+    const validTopics = ['mitch', 'cats']
+    
+    
+    if(!validSortBys.includes(sort_by)) {
+        return Promise.reject({status: 400, msg: "Bad request"});
+    }
+    
+    if(!validOrderBys.includes(order_by)) {
+        return Promise.reject({status: 400, msg: "Bad request"});
+    }
```

Comment:
Imlike that your error object here is nice and generic - you could combine these conditions with an || and just return a single Promise.reject